docker-compose.yml

```diff
@@ -114,6 +134,11 @@ services:
     command: ["webapp", "--dev"]
     ports:
       - "8000:8000"
+    develop:
+      watch:
+        - action: sync
+          path: .
+          target: /app
```

Comment:
This includes the sync action which will copy files from the host to the container.

There's also a rebuild action which I think we want to use to run manage.py collectstatic and esbuild when js and css files change.

We only need this rebuild on the webapp service because that's the only one using js and css files.